### PR TITLE
Fix the presence change

### PR DESCRIPTION
**Pull request by WalshyDev**
Make sure the setPresence call has the right parameters so that it does update correctly :)

### CODE REVIEW

File: src/main/java/net/dv8tion/jda/core/managers/impl/PresenceImpl.java [hunk 1]
```diff
@@ -86,19 +86,19 @@ public boolean isIdle()
     @Override
     public void setStatus(OnlineStatus status)
     {
-        setPresence(game, idle);
+        setPresence(game, stats);
```
Comment (binaryoverload) [inline src/main/java/net/dv8tion/jda/core/managers/impl/PresenceImpl.java]: Woopsie @WalshyDev I think this is meant to be "status" not "stats"

**Comment by kantenkugel** [comment]
code for reference:

https://github.com/DV8FromTheWorld/JDA/blob/master/src/main/java/net/dv8tion/jda/core/managers/impl/PresenceImpl.java#L92-L96

**Comment by binaryoverload** [comment]
#483 @WalshyDev You got ninjad

**Comment by WalshyDev** [comment]
@binaryoverload is it a ninja if that one is older? Also that doesn't fix the setStatus ;)

**Comment by binaryoverload** [comment]
@WalshyDev It's not older xD He opened it after yours

**Comment by kantenkugel** [comment]
Whoops... thought this was a Issue at first
Gonna drop my PR then...
Can you fix the setStatus one to use the 3-arg setStatus?

**Pull request merged**